R/write.R

```diff
@@ -1059,6 +1097,7 @@ write_data <- function(
     remove_cell_style = FALSE,
     na.strings        = na_strings(),
     inline_strings    = TRUE,
+    enforce           = FALSE,
```

Comment:
Adding another argument everywhere seems a bit verbose..

I don't know if this is good, but possibly via options is better? or 

adding it to user facing function save in temp env var, then apply it where needed?

Probably not a good suggestion

Reply:
If I understand you correctly: I made it an argument, because with the option you have to actively set it if needed and it didn’t work with fill either way.